### PR TITLE
fix: [#455] Session same_site config not working

### PR DESCRIPTION
**Pull request by kkumar-gcc**
## 📑 Description

Closes https://github.com/goravel/goravel/issues/455

@coderabbitai summary

## ✅ Checks

- [ ] Added test cases for my code

### CODE REVIEW

File: context_response.go [hunk 1]
```diff
@@ -24,6 +24,17 @@ func (r *ContextResponse) Cookie(cookie contractshttp.Cookie) contractshttp.Cont
...
 	}
+
+	sameSiteOptions := map[string]http.SameSite{
+		"strict": http.SameSiteStrictMode,
+		"lax":    http.SameSiteLaxMode,
+		"none":   http.SameSiteNoneMode,
+	}
+
+	if cookie.SameSite != "" {
+		r.instance.SetSameSite(sameSiteOptions[cookie.SameSite])
```
Comment (hwbrzzl) [inline context_response.go:35]: Check if the key exists.

**Review by devhaozi — APPROVED** [review]
LGTM

**Comment by devhaozi** [comment]
Does fiber need this change?
And please rebase it to master.

**Comment by kkumar-gcc** [comment]
> Does fiber need this fix?

No

**Pull request merged**